File: 6TH_SEMESTER/SWE 4604 - SOFTWARE TESTING AND QUALITY ASSURANCE LAB/6/Lab06CSharp/TestProject1/ProfilePage.cs
using OpenQA.Selenium;


namespace Lab06
{
    public class ProfilePage : BaseTest
    {
        [Fact]
        public void UpdateProfileInformationPhoneNumber()
        {

            driver.Navigate().GoToUrl("https://practicesoftwaretesting.com/auth/login");


            var email = driver.FindElement(By.Id("email"));
            email.SendKeys("customer@practicesoftwaretesting.com"); // Retrieved from the PDF
            var pass = driver.FindElement(By.Id("password"));
            pass.SendKeys("welcome01"); // retrieved from the PDF
            var submitbtn = driver.FindElement(By.ClassName("btnSubmit"));
            submitbtn.Click();


            var profile = driver.FindElement(By.CssSelector("a[data-test='nav-profile']"));
            profile.Click();


            var phoneInput = driver.FindElement(By.Id("phone"));
            phoneInput.SendKeys("210042112");


        }
    }
}

File: 6TH_SEMESTER/SWE 4604 - SOFTWARE TESTING AND QUALITY ASSURANCE LAB/6/testinglab06/src/tests/ProfilePage.java
package tests;

import org.junit.jupiter.api.Test;
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;

//210042112

public class ProfilePage extends BaseTest{

    @Test
    public void UpdateProfileInformationPhoneNumber()
    {
        driver.get("https://practicesoftwaretesting.com/auth/login");

        WebElement email = driver.findElement(By.id("email"));
        email.sendKeys("customer@practicesoftwaretesting.com");//taken from the pdf provided

        WebElement password = driver.findElement(By.id("password"));
        password.sendKeys("welcome01");//taken from the pdf provided

        WebElement loginbtn = driver.findElement(By.className("btnSubmit"));
        loginbtn.click();

        WebElement profile = driver.findElement(By.cssSelector("a[data-test='nav-profile']"));
        profile.click();

        WebElement phone =  driver.findElement(By.id("phone"));
        phone.sendKeys("210042112");

    }
}
// -_- N4M154 -_-